public/html/fragments/blog/ai-business-automation.html

```diff
@@ -6,6 +6,7 @@ <h1>AI in Sales: Why Your Team Needs a Smart Assistant, Not a Replacement</h1>
         <span class="post-date">Posted on: 2025-03-09</span>
         <span class="post-tags">Tags: AI, Sales, Automation, Business</span>
         <span class="post-author">By: BitsOf Team</span>
+        <span class="post-keywords">Keywords: AI sales automation, human-AI collaboration, sales efficiency, business automation, AI assistants, sales productivity</span>
```

Comment:
fragments don't exist anymore. Please make sure you are on the right branch of the code. You need to be off the current main.